its/src/test/java/com/sonar/it/csharp/MetricsTest.java

```diff
@@ -59,26 +59,14 @@ private static RuleChain getRuleChain() {
         @Override
         protected void before() throws Throwable {
           TestUtils.reset(ORCHESTRATOR);
-
-          Path projectDir = Tests.projectDir(temp, "MetricsTest");
-
-          ScannerForMSBuild beginStep = TestUtils.createBeginStep("MetricsTest", projectDir)
-            .setProfile("no_rule")
-            // Without that, the MetricsTest project is considered as a Test project :)
```

Comment:
Can you preserve this comment? I find it useful. Similar one is in `NoSonarTest.java`